frontend/webpack.config.js

```diff
@@ -12,7 +12,7 @@ var path = require('path');
 var config = {
     entry: path.join(__dirname, 'app', 'index.js'),
     output: {
-        path: path.resolve(__dirname, 'public'),
+        path: '/var/www/promis', // TODO: do we need to parametrise this?
```

Comment:
I mean if this script is located in `/var/www/promis`, then `__dirname === '/var/www/promis'` and `path.join(__dirname, 'public'` should be just fine